portfolio/src/main/webapp/scripts/server-dev.js

```diff
@@ -12,21 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-var answer_idx = 0;
-
-function getAnswer() {
-  fetch('/data')
+/**
+ * Fetches the previously entered comments from the server and inserts each
+ * comment as a list item of the 'comments' <ul> element.
+ */
+function getCommentsThread() {
+  fetch('/comment-data')
       .then(response => response.json())
-      .then(answers => {
-        document.getElementById('answer-container').innerText =
-            answers[(answer_idx++) % answers.length];
-        if (answer_idx >= Number.MAX_SAFE_INTEGER) {
-          answer_idx = 0;
-        }
-      })
-      .catch(err => {
-        console.log('Error: ' + err);
-        document.getElementById('answer-container').innerText =
-            'Something went wrong.';
-      });
+      .then((commentList) => {
+        const commentThread = document.getElementById('comments-thread');
+        commentList.forEach((comment) => {
+          commentThread.appendChild(createListElement(comment));
+        })
```

Comment:
What happens if the comment contains HTML? For instance, an XSS attack containing a <script> tag. https://en.wikipedia.org/wiki/Cross-site_scripting

Reply:
I actually just saw this as one of the "next steps" ideas after step 6 is complete. Would you prefer I implement this now or once I have completed all of the guided steps?

Reply:
Created issue #15 .